README.md

```diff
@@ -39,11 +56,54 @@ We provide a `Dockerfile` to build a Docker image for PyMC-Marketing so that is
 
 Leverage our Bayesian MMM API to tailor your marketing strategies effectively. Based on the research [Jin, Yuxue, et al. “Bayesian methods for media mix modeling with carryover and shape effects.” (2017)](https://research.google/pubs/pub46001/),  and integrating the expertise from core PyMC developers, our API provides:
 
+- **Custom Priors and Likelihoods**: Tailor your model to your specific business needs by including domain knowledge via prior distributions.
 - **Adstock Transformation**: Optimize the carry-over effects in your marketing channels.
 - **Saturation Effects**: Understand the diminishing returns in media investments.
+- **Visualization and Model Diagnostics**: Get a comprehensive view of your model's performance and insights.
+- **Out-of-sample Predictions**: Forecast future marketing performance with credible intervals. Use this for simulations and scenario planning.
 - **Budget Optimization**: Allocate your marketing spend efficiently across various channels for maximum ROI.
 - **Experiment Calibration**: Fine-tune your model based on empirical experiments for a more unified view of marketing.
 
+### MMM Quickstart
+
+```python
+import pandas as pd
+from pymc_marketing.mmm import DelayedSaturatedMMM
+
```

Comment:
Nitpick: prefer the look of single space between inport and code in readme file 😅

Reply:
right! fixed in https://github.com/pymc-labs/pymc-marketing/pull/653/commits/489510c17611fb4424ec759a980d1f21fbff1e8d